src/api/metrics.rs

```diff
@@ -1,28 +1,30 @@
-use crate::{cli::Config, BlockfrostError};
+use crate::BlockfrostError;
 use axum::response::{Extension, IntoResponse};
 use metrics::{describe_counter, describe_gauge, gauge};
 use metrics_exporter_prometheus::{PrometheusBuilder, PrometheusHandle};
 use std::sync::Arc;
 use tokio::sync::RwLock;
 
 pub async fn route(
-    Extension(prometheus_handle): Extension<Arc<RwLock<PrometheusHandle>>>,
-    Extension(config): Extension<Arc<Config>>,
+    Extension(prometheus_handle): Extension<Option<Arc<RwLock<PrometheusHandle>>>>,
 ) -> Result<impl IntoResponse, BlockfrostError> {
-    if !config.metrics {
-        return Err(BlockfrostError::not_found());
+    match prometheus_handle {
```

Comment:
I have a fix for this, I am working on the tests. This is good for as a quick fix.

Reply:
Ah, right! I had to include this to be able to see `GET /metrics` for my investigation, but your fix in:
* #182 

… is better. I tried to keep this one minimal in terms of LOC.

I'll resolve the conflict as soon as your PR gets merged.

Reply:
No worries, whoever gets there first!